### ISSUE TITLE
tmux support

### ISSUE THREAD

**Issue by toejough** [franciscolourenco/done#49]
AFAICT, this doesn't work with tmux.

OS: MacOS 10.14.5
Terminal: iTerm2 3.3.0beta5
tmux: 2.9a
fish: 3.0.2

without tmux, this works as expected.  With tmux, I get no notifications.

**Comment by franciscolourenco** [comment]
@toejough it does work for me. Are you logging into a remote server or local tmux server?

**Comment by toejough** [comment]
Local

**Comment by franciscolourenco** [comment]
@toejough The following steps work for me using the same versions. They don't work for you?

1. Type `sleep 15`
2. Switch to another application
3. Wait 15 seconds
4. Receive notification.

**Comment by toejough** [comment]
No...  :-/

I appreciate your responsiveness & repro attempts!  I'm not sure what the difference could be now.

I've:

* reinstalled `terminal-notifier` and `done`
* closed tmux & iterm
* opened a new iterm
* run `sleep 5` and clicked off to another app window
* gotten a notification after 5s.
* moved my `.tmux.conf` so I had vanilla tmux config
* started a local tmux session with `tmux`
* run `sleep 5` and clicked off to another app window
* no notification.

Is there any log I can look at to debug this further?

**Comment by toejough** [comment]
(I also tried `sleep 15`, to be sure it's not about edge cases)

**Comment by toejough** [comment]
FWIW, `terminal-notifier -message YO` shows the expected notification both in iterm and in tmux (even with my custom .tmux.conf).

**Comment by toejough** [comment]
...I'm no longer having this issue.  I did uninstall terminal-notifier, and then a few days later this plugin started working and it caught me totally off guard.

I reinstalled terminal-notifier, and everything still works.  I'm very confused, but I'll close the issue.

**Comment by toejough** [comment]
Also, now that it's working for me, I find this *really* cool.  Thank you for making it!

**Comment by franciscolourenco** [comment]
Thanks for the update!

**Comment by gpanders** [comment]
Unfortunately I'm having a similar issue:

1. Works when not in tmux
2. Does not work in tmux
3. Works when `terminal-notifier` is not installed (uses `osascript` instead)

I have tried uninstalling and re-installing `terminal-notifier` but this did not change anything.

FWIW, I've noticed that the following
```console
terminal-notifier -message "Hi" -title "Foo" -sender "tmux" -activate "$__done_initial_window_id"
```
does not work, but omitting the `-sender` argument does work:
```console
terminal-notifier -message "Hi" -title "Foo" -activate "$__done_initial_window_id"
```
So I think there might be a problem with `-sender "tmux"`.
